pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
If there is  a fixed maxderivs, it should probably be based on some numerical reason why going beyond that is likely to cause bad results and users should be prevented from doing so.